lit_parrot/speed_monitor.py

```diff
@@ -57,17 +57,16 @@
     "quadro rtx 5000": {"32-true": 11.2e12, "16-true": 89.2e12, "16-mixed": 89.2e12},
 }
 
-TPU_AVAILABLE_FLOPS={
+TPU_AVAILABLE_FLOPS = {
     # flop count for each TPU generation is the same for all precisions
     # since bfloat16 precision is always used for performing matrix operations
     # for more info: https://cloud.google.com/tpu/docs/bfloat16#choosing_bfloat16
-
     # source: https://arxiv.org/pdf/1907.10701.pdf
-    "V2": 45e12,
+    "v2": 45e12,
```

Comment:
from the output in https://github.com/Lightning-AI/lit-gpt/pull/186#issuecomment-1602963959, shouldn't the `v` be `V`?

Reply:
I added .lower() to be case invariant

Reply:
Ah thanks for pointing this out, I didn't notice